docs/operators/omit.md

```diff
@@ -0,0 +1,49 @@
+---
+title: "@omit"
+---
+
+Within a GraphQL schema, the **@omit** operator excludes fields or nodes from the generated schema, making them inaccessible through the GraphQL API. This operator is useful for hiding sensitive information or simplifying your API by removing unnecessary fields.
+
+## How it works
+
+When applied to a field or node, the **@omit** operator instructs the Tailcall not to include that field or node in the schema. This means that clients cannot query or mutate data in those fields.
+
+## Example
+
+Consider a scenario where you have a `User` type with an embedded `Address` type. If you want to exclude the `Address` type from the schema to simplify the API, you can use the **@omit** operator:
+
+
+```graphql showLineNumbers
+type Address {
+  city: String
+  street: String
+}
+
+type User {
+  name: String
+  address: Address @omit
+}
+```
+
+
+In this example, the `address` field will not be accessible or visible through the GraphQL API.
+
+## Comparison with @modify(omit: true)
```

Comment:
I would reword this section to imply that `@omit` is a alias to `@modify(omit: true)` and build on that instead of saying they are different.